doc/_resources/templates/document.html

```diff
@@ -8,15 +8,14 @@
 
 {{define "versionSelector"}}
     <div id="version-selector" class="dropdown version-dropdown ml-2">
+        {{/* Update these after each release. */}} {{$previousReleaseRevspec := "v3.18.0"}} {{$previousReleaseVersion := "3.18"}} {{$currentReleaseRevspec := "v3.19.1"}} {{$currentReleaseVersion := "3.19"}} {{$nextReleaseVersion := "main"}}
         <button class="btn btn-outline dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            {{.ContentVersion}}{{/* If no version is specified we are on the main branch */}}{{if eq .ContentVersion ""}}main{{end}}
+          {{.ContentVersion}}{{/* If no version is specified we are on the currentReleaseVersion */}}{{if eq .ContentVersion ""}}{{$currentReleaseVersion}}{{end}}
         </button>
         <details-menu aria-label="Sourcegraph version" class="dropdown-menu m-0 p-0" aria-labelledby="Version Dropdown">
-
-            {{/* Update these after each release. */}} {{$previousReleaseRevspec := "v3.18.0"}} {{$previousReleaseVersion := "3.18"}} {{$currentReleaseRevspec := "v3.19.1"}} {{$currentReleaseVersion := "3.19"}} {{$nextReleaseVersion := "main"}}
-            <a rel="nofollow" class="dropdown-item {{if eq .ContentVersion " "}}active{{end}}" href="/{{.ContentPagePath}}">main</a>
-            <a rel="nofollow" class=" dropdown-item {{if eq $currentReleaseVersion .ContentVersion " "}}active{{end}}" href="/@{{$currentReleaseVersion}}/{{.ContentPagePath}}">{{$currentReleaseVersion}}<span class="current-branch ml-1">current</span></a>
+            <a rel="nofollow" class=" dropdown-item {{if eq .ContentVersion " "}}active{{end}}" href="/@{{$currentReleaseVersion}}/{{.ContentPagePath}}">{{$currentReleaseVersion}}<span class="current-branch ml-1">current</span></a>
             <a rel="nofollow" class=" dropdown-item {{if eq $previousReleaseVersion .ContentVersion " "}}active{{end}}" href="/@{{$previousReleaseVersion}}/{{.ContentPagePath}}">{{$previousReleaseVersion}}</a>
+            <a rel="nofollow" class="dropdown-item {{if eq $nextReleaseVersion .ContentVersion " "}}active{{end}}" href="/{{.ContentPagePath}}">main</a>
```

Comment:
For clarity, maybe we could have the UI could show `Unreleased` instead? It might not be immediately obvious what `main` means

```suggestion
            <a rel="nofollow" class="dropdown-item {{if eq $nextReleaseVersion .ContentVersion " "}}active{{end}}" href="/{{.ContentPagePath}}">Unreleased</a>
```

Reply:
That's good! Thanks!